cypress/e2e/mapp-tracking.cy.ts

```diff
@@ -0,0 +1,31 @@
+describe("Mapp Tracking", () => {
+  it("tracks page views", () => {
+    const customerId = "1234";
+
+    // Mapp will not perform requests if wt_r cookie is set so clear it before
+    // our test and also use cookie debugging to help understand the process.
+    cy.clearCookies();
+    cy.drupalLogin();
+    cy.visit("/admin/config/system/dpl-mapp")
+      .get('[name="id"]')
+      .clear()
+      .type(customerId)
+      .parent()
+      .get('[value="Gem indstillinger"]')
```

Comment:
With all the talk and changes we had lately, regarding admin language, would this still work with danish text assertions?

Reply:
It will not. If the admin language changes for the admin user then these texts will have to be updated accordingly.

Reply:
ok